python/triton/language/semantic.py

```diff
@@ -808,7 +808,7 @@ def store(ptr: tl.tensor,
         raise ValueError("Pointer argument of store instruction is " + ptr.type.__repr__())
     if ptr.type.is_block():
         val = broadcast_impl_shape(val, ptr.type.get_block_shapes(), builder)
-    if mask:
+    if mask and ptr.type.is_block():
```

Comment:
When calling `tl.load` with a scalar address and a mask, here will fail due to `ptr.type` has no attribute called `is_block`.

Reply:
> When calling tl.load with a scalar address and a mask, here will fail due to ptr.type has no attribute called is_block.

I don't understand. `is_block` is a method right?

Reply:
Also, can you add a test?

Reply:
Sorry, a mistake, the error is "ptr.type has no attribute called get_block_shapes".

> Also, can you add a test?
Sure